paper-menu-button.html

```diff
@@ -90,11 +90,14 @@
       margin-top: 20px;
     }
 
-    paper-material {
+    iron-dropdown {
+      @apply(--paper-menu-button-dropdown);
+    }
+
+    .dropdown-content {
       border-radius: 2px;
       background-color: var(--paper-menu-button-dropdown-background, --primary-background-color);
-
-      @apply(--paper-menu-button-dropdown);
+      @apply(--paper-menu-button-content);
```

Comment:
I think it may be important to allow for a style hook here, since it would otherwise not be possible to override the border-radius.